proto/cosmos/group/v1beta1/tx.proto

```diff
@@ -245,17 +245,17 @@ message MsgCreateProposal {
   // metadata is any arbitrary metadata to attached to the proposal.
   bytes metadata = 3;
 
-  // msgs is a list of Msgs that will be executed if the proposal passes.
-  repeated google.protobuf.Any msgs = 4;
+  // messages is a list of `sdk.Msg`s that will be executed if the proposal passes.
+  repeated google.protobuf.Any messages = 4;
 
   // exec defines the mode of execution of the proposal,
   // whether it should be executed immediately on creation or not.
   // If so, proposers signatures are considered as Yes votes.
   Exec exec = 5;
```

Comment:
Just so I understand this correctly, if this is true, the application will create the proposal count up the votes from the proposers and execute the messages if the proposal already passes?

Reply:
exactly.

https://github.com/cosmos/cosmos-sdk/blob/b9c7fd11d356a7fb51d76b8a7f79425fca02e750/x/group/keeper/msg_server.go#L460-L479